node/packages/aws-lambda-otel-extension/external/otel-extension-external/otel-payloads.js

```diff
@@ -149,6 +148,16 @@ const createLogPayload = (eventData, logs) => {
     resourceAtt[attribute.key] = Object.values(attribute.value || {})[0];
   }
 
+  return {
+    resourceAtt,
+    metricsAtt,
+  };
+};
+
+const createLogPayload = (eventData, logs) => {
```

Comment:
Same here and in other places in this file (?)

Reply:
Another good call. I think there is one other place in this file where this needs to change I will be sure to change that as well 😎

Reply:
This should be done ✅